distribution/manifest.json

```diff
@@ -33,6 +34,7 @@
 		"page": "options.html"
 	},
 	"background": {
+		"persistent": false,
```

Comment:
This is required for iOS Safari.

Reply:
😬 didn't we remove it recently for something else? Or maybe that's been addressed? I think we discussed this within the last 6 months.

Reply:
I know we talked about it, but I don't remember the outcome. We probably just forgot.

Reply:
👍 

I said this:

> But actually Firefox doesn’t support persistent:false at all so we’re good at this point